app/src/main/kotlin/org/eclipse/kuksa/testapp/databroker/view/DataBrokerView.kt

```diff
@@ -555,26 +556,31 @@ fun DataBrokerOutput(viewModel: OutputViewModel, modifier: Modifier = Modifier)
     val shape = RoundedCornerShape(20.dp, 20.dp, 0.dp, 0.dp)
     val scrollState = rememberScrollState(0)
 
+    val output = viewModel.output
+
     Surface(
         modifier = modifier.height(500.dp),
         color = MaterialTheme.colorScheme.primary,
         shape = shape,
     ) {
         Column(modifier = Modifier.verticalScroll(scrollState)) {
             Headline(name = "Output", color = Color.White)
-            Text(
-                modifier = Modifier
-                    .fillMaxHeight()
-                    .fillMaxWidth()
-                    .padding(start = DefaultElementPadding, end = DefaultElementPadding),
-                text = viewModel.output,
-                textAlign = TextAlign.Start,
-                onTextLayout = {
-                    scope.launch {
-                        scrollState.animateScrollTo(scrollState.maxValue)
-                    }
-                },
-            )
+            output.forEach { outputElement ->
+                Text(
+                    modifier = Modifier
+                        .fillMaxHeight()
+                        .fillMaxWidth()
+                        .padding(start = DefaultElementPadding, end = DefaultElementPadding),
+                    text = outputElement,
+                    fontSize = 14.sp,
```

Comment:
Default fontSize is 16.sp - however this might also vary depending on the system settings